docs/server/production-deploy.md

```diff
@@ -68,20 +68,30 @@ Every shard is low contention by design and it is very difficult to oversubscrib
 With that said, here are some guidelines to some common bottlenecks:
 
 - **Database**. The vast majority of the time the database will be the bottleneck. **We highly recommend setting alerts on `ScheduleToStart` latency** to look out for this. Also check if your database connection getting saturated.
-- **Internal services**. The next layer will be scaling the 4 internal services of Temporal ([Frontend, Matching, History, and Worker](https://docs.temporal.io/docs/server-architecture/)). Monitor each accordingly. The Frontend service is more CPU bound, whereas the History and Matching services require more memory.
+- **Internal services**. The next layer will be scaling the 4 internal services of Temporal ([Frontend, Matching, History, and Worker](https://docs.temporal.io/docs/server-architecture/)).
+  Monitor each accordingly. The Frontend service is more CPU bound, whereas the History and Matching services require more memory.
```

Comment:
Any way to add more info on how to monitor each accordingly?

Reply:
i'd love to but dont have specific ideas apart from literally watching CPU graphs